docs/install/ubuntu_setup.md

```diff
@@ -180,58 +180,16 @@ You might want to add this command to your ```~/.bashrc``` file. If you do, you
 
 For more details about installing and using MXNet with Julia, see the [MXNet Julia documentation](http://dmlc.ml/MXNet.jl/latest/user-guide/install/).
 
-### Install the MXNet Package for Scala
-There are two ways to install the MXNet package for Scala:
 
-* Use the prebuilt binary package
+## Install the MXNet Package for Scala
 
-* Build the library from source code
+To use the MXNet-Scala package, you can acquire the Maven package as a dependency.
 
-#### Use the Prebuilt Binary Package
-For Linux users, MXNet provides prebuilt binary packages that support computers with either GPU or CPU processors. To download and build these packages using ```Maven```, change the ```artifactId``` in the following Maven dependency to match your architecture:
+Further information is in the [MXNet-Scala Setup Instructions](./scala_setup.md).
```

Comment:
I am still not clear if the maven-dependency xml snippet would appear on the main install page?

Reply:
@nswampy Please use the links in the main comment above. The first two show what you're looking for... click between CPU and GPU and Mac and Linux to see the variations.

Reply:
Thanks missed your anime 👍